src/styles/elements/SectionHeader/SectionHeader.js

```diff
@@ -3,7 +3,6 @@ import styled from 'styled-components';
 import { colors, spacing, breakPoints } from 'styles/base';
 
 const SectionHeader = styled.h2`
-  margin-top: ${spacing.xxxxlSize};
```

Comment:
to be hones, those spacings are quite shit. I am wondering when you come to:
`xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxSize`
Why not just use value here? It seems to be not very scalable.